react/components/FilterSidebar.js

```diff
@@ -148,7 +148,7 @@ const FilterSidebar = ({
             </Button>
           </div>
           <div
-            className={`${handles.filterApplytButtonWrapper} bottom-0 fr w-50 pr4 pl2`}
+            className={`${handles.filterApplyButtonWrapper} bottom-0 fr w-50 pr4 pl2`}
```

Comment:
How long does this handle exists? If that's something old, we might break someone which is using the name with typo

Reply:
It is new